### PR TITLE
Added requests for next azure release

### PR DESCRIPTION
**Pull request by whites11**
Enforce kubernetes 1.17.12, external-dns 1.5.0 and newest flatcar for next azure release

### CODE REVIEW

File: azure/requests.yaml [hunk 1]
```diff
@@ -14,9 +14,6 @@ releases:
         - name: containerlinux
           version: ">= 2512.4.0"
           issue: https://github.com/giantswarm/giantswarm/issues/13150
-        - name: external-dns
```
Comment (fiunchinho) [inline azure/requests.yaml:17]: Not sure how this works so bear with me. Why is this removed?

Comment (whites11) [reply azure/requests.yaml:17]: It was added in the wrong section, for clusters < 13.0.0.
We're not going to release any new release < 13.0.0

**Review by tuommaki — APPROVED** [review]
